libvmi/json_profiles/volatility_ist.h

```diff
@@ -53,4 +53,16 @@ static inline const char* volatility_get_os_type(vmi_instance_t vmi)
 
 #endif
 
+static inline status_t
+volatility_profile_bitfield_offset_and_size(
+    __attribute__((__unused__)) json_object *json,
+    __attribute__((__unused__)) const char *symbol,
+    __attribute__((__unused__)) const char *subsymbol,
+    __attribute__((__unused__)) addr_t *rva,
+    __attribute__((__unused__)) size_t *start_bit,
+    __attribute__((__unused__)) size_t *end_bit)
+{
+    return VMI_FAILURE;
```

Comment:
This is unfortunately a no-go. We are moving away from Rekall as it is unmaintained. Please implement this.

Reply:
It would be hard to me to test because I don't use volatility yet (and not going in near feature). Though I could try to write some code.

Reply:
The volatility json is very similar to the rekall one. It really shouldn't be difficult.

Reply:
done